portfolio/src/main/webapp/blogs.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html lang="en-US">
+  <head>
+    <meta charset="UTF-8">
+	<meta name="Feiyang (Kathy) Yu's Blog Posts" content="Hidden talent blog posts of Feiyang (Kathy) Yu">
+	<link rel="stylesheet" type="text/css" href="/style.css">
+    <script src="/script.js"></script>
+	<title>Kathy's Blog Posts</title>
+  </head>
+  
+  <body>
+    <!--- Main section contains the blog post content -->
```

Comment:
contents

Reply:
Just to clarify, by "contents" do you mean that I should add the contents class attribute? Thank you!

Reply:
Sorry, it was more of a language nitpick. I meant change "blog post content" to "blog post contents" :)